src/utils/sykmeldinger/sykmeldingUtils.ts

```diff
@@ -265,7 +262,8 @@ export const sykmeldingerGruppertEtterVirksomhet = (
 ): SykmeldingerPerVirksomhet => {
   return sykmeldinger.reduce((memo: SykmeldingerPerVirksomhet, sykmelding) => {
     const virksomhetsnummer =
-      sykmelding.mottakendeArbeidsgiver?.virksomhetsnummer;
+      sykmelding.mottakendeArbeidsgiver?.virksomhetsnummer ??
+      "Ukjent virksomhet";
```

Comment:
For at de sykmeldingene som tilhører arbeidsgiver, men ikke sendt enda (`NY` eller `APEN`) skal dukke opp, må vi gi de en nøkkel i sorteringen. Denne vises ikke noe sted, bare brukes for sortering på arbeidsgiver.